site/content/docs/cli/constructs-reference.md

```diff
@@ -301,29 +284,86 @@ new MultiStepCheck('multistep-check-1', {
 - `code`: an object with either an `entrypoint` property that points to `.spec.js|ts` file, or a `content` property with
 raw JavaScript / TypeScript as a string.
 
+## Uptime Monitors
+
+Use Uptime monitors to regularly check if a website, port, or service is available and responding as expected.
+
+## `UrlMonitor`
+Use URL monitors to track basic availability and HTTP status codes of your services. The example below:
+
+- Defined basic monitor properties like `name`, `activated` etc.
+- Sends a `GET` rquest to a target `url`.
+- Includes an assertion that expects an HTTP 200 OK response.
+- Sets thresholds for when the check should be marked as degraded or failing.
+
+The file hierarchy looks as follows:
+
+```
+├── __monitors__
+│   ├── hello-url.monitor.ts
+```
+
+```ts {title="hello-url.monitor.ts"}
+import { UrlMonitor, AssertionBuilder } from 'checkly/constructs'
+import * as path from 'path'
+
+new UrlMonitor('hello-url-1', {
+  name: 'Hello URL',
+  activated: true,
+  maxResponseTime: 10000,
+  degradedResponseTime: 5000,
+  request: {
+    method: 'GET',
+    url: 'https://httpbin.org/get',
+    skipSSL: false,
+    followRedirects: true,
+    assertions: [
+        AssertionBuilder.statusCode().equals(200),
+    ]
+  }
+})
```

Comment:
There is an extra `path` import for no reason, and `UrlMonitor` should use `UrlAssertionBuilder`, which limits the available assertion methods to those supported by `UrlMonitor`.

```suggestion
import { UrlMonitor, UrlAssertionBuilder } from 'checkly/constructs'

new UrlMonitor('hello-url-1', {
  name: 'Hello URL',
  activated: true,
  maxResponseTime: 10000,
  degradedResponseTime: 5000,
  request: {
    method: 'GET',
    url: 'https://httpbin.org/get',
    skipSSL: false,
    followRedirects: true,
    assertions: [
        UrlAssertionBuilder.statusCode().equals(200),
    ]
  }
})
```